app/javascript/packs/initializers/initializeSettings.js

```diff
@@ -0,0 +1,65 @@
+import { timestampToLocalDateTime } from '@utilities/localDateTime';
+
+export class CopyOrgSecret {
+  static copyToClipboardListener() {
+    const settingsOrgSecretInput = document.getElementById(
+      'settings-org-secret',
+    );
+    if (settingsOrgSecretInput === null) return;
+
+    const { value } = settingsOrgSecretInput;
+    return window.Forem.Runtime.copyToClipboard(value).then(() => {
+      // Show the confirmation message
+      document.getElementById('copy-text-announcer').classList.remove('hidden');
+    });
+  }
+
+  static initialize() {
+    document
+      .getElementById('settings-org-secret-copy-btn')
+      ?.addEventListener('click', this.copyToClipboardListener);
+  }
+}
+
+export class RssFetchTime {
+  static initialize() {
+    // shows RSS fetch time in local time
+    const timeNode = document.getElementById('rss-fetch-time');
+    if (timeNode) {
+      const timeStamp = timeNode.getAttribute('datetime');
```

Comment:
To test that initializers are all hooked up as they should be we typically use an end-to-end test like [this](https://github.com/forem/forem/blob/8335a37315d3aa605ac34e6e98d7c79c0cfac3b7/cypress/e2e/seededFlows/profileFlows/userProfile.spec.js) or [this](https://github.com/forem/forem/blob/8335a37315d3aa605ac34e6e98d7c79c0cfac3b7/cypress/e2e/seededFlows/articleFlows/showFullDatesOnHover.spec.js), which should help with the test coverage as well